packages/common/test/discussions/fetch.test.ts

```diff
@@ -39,6 +41,10 @@ describe("discussions fetch:", () => {
       const getItemDataSpy = spyOn(portalModule, "getItemData").and.returnValue(
         Promise.resolve(DISCUSSION_DATA)
       );
+      const fetchSettingsSpy = spyOn(settingUtils, "fetchSetting").and.callFake(
```

Comment:
Similar as above, `fetchSettingSpy` returns a `Promise<IEntitySetting>` while this spy returns a `Partial<IEntitySetting>` (missing item id). `fetchSettingsSpy` is an unused const, we should check that it's called with what we'd expect it to be called with then verify that `chk.entitySettingsId` and `chk.discussionSettings` are set to the expected value